app/javascript/sdk/bubbleHelpers.js

```diff
@@ -61,6 +61,9 @@ export const createBubbleHolder = hideMessageBubble => {
     addClasses(bubbleHolder, 'woot-hidden');
   }
   addClasses(bubbleHolder, 'woot--bubble-holder');
+
+  bubbleHolder.style = 'overflow: hidden;';
```

Comment:
It would be best to add this style to the `woot--bubble-holder` class, instead of adding it as a style